Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -301,5 +298,21 @@ class DefaultBrowserOnboardingViewController: UIViewController {
         TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .goToSettingsDefaultBrowserOnboarding)
         LeanPlumClient.shared.track(event: .goToSettingsDefaultBrowserOnboarding)
     }
+  
+    // Theme
+    @objc func updateTheme() {
+        self.view.backgroundColor = fxBackgroundThemeColour
+        self.imageText.textColor = fxTextThemeColour
+        self.titleLabel.textColor = fxTextThemeColour
+        self.descriptionText.textColor = fxTextThemeColour
+        self.descriptionLabel1.textColor = fxTextThemeColour
+        self.descriptionLabel2.textColor = fxTextThemeColour
+        self.descriptionLabel3.textColor = fxTextThemeColour
+        viewModel.refreshModelImage()
+        self.topImageView.image = viewModel.model?.titleImage
```

Comment:
Added these two lines to reflect correct dark them image 
<img width="252" alt="Screen Shot 2021-02-24 at 5 39 21 PM" src="https://user-images.githubusercontent.com/8919439/109075507-3aae4100-76c7-11eb-8ad6-766ac7d724e4.png">